### PR TITLE
Updating Outdated Mac Support

### PR DESCRIPTION
**Pull request by TokyoScarab**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2459

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/plan-your-deployment/clients-and-devices/meetings-clients.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ description: "Summary: IT Professionals should review the support requirements f
 ---
 
 # Plan for Meetings clients (Web App and Meetings App)
+  
+ >[!IMPORTANT]
+ >As of April 5th, 2019, the Skype for Business Web App and Skype Meetings App will be replaced by the Skype for Business desktop app for Mac users. For more information, please see [Skype for Business desktop app on Mac to replace Skype Meetings App (web)](https://techcommunity.microsoft.com/t5/Skype-for-Business-Blog/Skype-for-Business-desktop-app-on-Mac-to-replace-Skype-Meetings/ba-p/357958).
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/clients-and-devices/meetings-clients.md]:
```suggestion
 >On April 5th, 2019, the Skype for Business Web App and Skype Meetings App have been replaced by the Skype for Business desktop app for Mac users. For more information, please see [Skype for Business desktop app on Mac to replace Skype Meetings App (web)](https://techcommunity.microsoft.com/t5/Skype-for-Business-Blog/Skype-for-Business-desktop-app-on-Mac-to-replace-Skype-Meetings/ba-p/357958).
```

Comment (nenonix) [reply Skype/SfbServer/plan-your-deployment/clients-and-devices/meetings-clients.md]: On....have been?  Suggest review is reviewed

**Comment by scanum** [comment]
@TokyoScarab  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tonysmit** [comment]
@scanum - I don't own this topic and can't merge it because I don't know anything about this area. @TokyoScarab - do you have confirmation from the PG on these changes? I'm going to add @jambirk to see if he knows anything here.

**Comment by jambirk** [comment]
@PhillipGarding is the main PG contact. Phillip, do you approve of these changes?

**Comment by PhillipGarding** [comment]
No, I don't approve these changes.  Why are you removing reference to the Mac app?  The transition that is being documented is SfB Web app (supported for Windows and Mac) ==> Skype Meetings App on Windows and SfB for Mac on Mac.  Both apps must be included in the description.

**Comment by LolaJacobsen** [comment]
Not merged per PM Phillip Garding.

**Comment by jambirk** [comment]
@PhillipGarding to answer your question, see comment thread here: https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2459

**Pull request closed without merging**